CalibPPS/TimingCalibration/python/ALCARECOPPSTimingCalib_Output_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+import copy
+
+OutALCARECOPPSTimingCalib_noDrop = cms.PSet(
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOPPSTimingCalib')
+    ),
+    outputCommands = cms.untracked.vstring(
+        'keep *_MEtoEDMConvertPPSTimingCalib_*_*',
+    )
+)
+
+OutALCARECOPPSTimingCalib = copy.deepcopy(OutALCARECOPPSTimingCalib_noDrop)
```

Comment:
please use clone()

Reply:
please remove also `import copy`

Reply:
Done in https://github.com/cms-sw/cmssw/pull/33215/commits/f9c1fcdef51e305ba980e8ce13e29ed61c2a5167